api-tests/dev_apis/crypto/test_c016/test_c016.c

```diff
@@ -82,11 +83,8 @@ int32_t psa_generate_key_test(caller_security_t caller)
         val->crypto_function(VAL_CRYPTO_GET_KEY_TYPE, &get_attributes, &get_key_type);
         TEST_ASSERT_EQUAL(get_key_type, check1[i].key_type, TEST_CHECKPOINT_NUM(5));
 
-        if (check1[i].attr_bits != 0)
-            TEST_ASSERT_EQUAL(get_attributes.bits, check1[i].attr_bits, TEST_CHECKPOINT_NUM(6));
-        else
-            TEST_ASSERT_EQUAL(get_attributes.bits, check1[i].expected_bit_length,
-                              TEST_CHECKPOINT_NUM(7));
+        val->crypto_function(VAL_CRYPTO_GET_KEY_BITS, &get_attributes, &get_key_bits);
+        TEST_ASSERT_EQUAL(get_key_bits, check1[i].expected_bit_length, TEST_CHECKPOINT_NUM(6));
 
         val->crypto_function(VAL_CRYPTO_GET_KEY_USAGE_FLAGS, &get_attributes, &get_key_usage);
         TEST_ASSERT_EQUAL(get_key_usage, check1[i].usage, TEST_CHECKPOINT_NUM(8));
```

Comment:
Please make sure `TEST_CHECKPOINT_NUM` are updated accordingly for all the tests_c002,16,50. It needs to continuous, rather than gaps...